packages/sources/genesis-volatility/src/adapter.ts

```diff
@@ -15,12 +16,13 @@ const daysConversion: Record<number, string> = {
   28: 'twentyEightDayIv',
 }
 
-const execute: ExecuteWithConfig<Config> = async (input, config) => {
-  const validator = new Validator(input, customParams)
+// TODO: Run tests with valid pro tier + API Key
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  const validator = new Validator(request, customParams)
   if (validator.error) throw validator.error
 
   const jobRunID = validator.validated.id
-  const url = 'https://app.pinkswantrading.com/graphql'
+  const url = config.api.baseURL
```

Comment:
Could spread this in. I think there will only ever be one API endpoint here.

Reply:
Not sure what you mean by spread this in here, could you elaborate please?